wiki/en/en-Software-Manual.md

```diff
@@ -72,7 +72,7 @@ pressing this button will end the session.
 
 ![Connect dialog](https://user-images.githubusercontent.com/20726856/97360965-31d76b00-189f-11eb-870b-f58be6e30f04.png)
 
-The Connection Setup window shows a list of available servers together with the number of occupants and the maximum number supported. Server operators can optionally list their servers by musical genre. Use the List dropdown to select a genre, click on the server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name. Permanent servers (those that have been listed for longer than 24 hours) are shown in bold.
+The Connection Setup window shows a list of available servers together with the number of occupants and the maximum number supported. Server operators can optionally list their servers by music genre. Use the List dropdown to select a genre, click on the server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name. Permanent servers (those that have been listed for longer than 24 hours) are shown in bold.
```

Comment:
It's not optional.  All servers have a server list.  Some of the server lists are genre-specific.  There could be location-specific ones.  But they're all on lists.  I think this needs explaining more clearly.

Reply:
This is the client's manual don't forget, so we're talking about their view of that list. But maybe we can just drop that line so it reads:

"... the maximum number supported. Use the List dropdown to select a genre, click on the server..."

Reply:
Rephrased it. See what you reckon.